src/docs/product/releases/index.mdx

```diff
@@ -8,70 +8,88 @@ redirect_from:
-3.  [Create Release and Associate Commits](#create-release)
-4.  [Tell Sentry When You Deploy a Release](#create-deploy)
+Setting up releases fully requires:
 
-### Configure Your SDK {#configure-sdk}
+1. [Configuring Your SDK](/platform-redirect/?next=/configuration/releases/) to include a release identifier, which is documented for each SDK
+2. [Notifying Sentry](#notify) of a release, which is documented on this page
 
-Including a release ID is documented as part of [configuring your SDK to provide release information](/platform-redirect/?next=/configuration/releases/).
+## Notify
 
-After configuring your SDK, you will see information about the release, such as new issues and regressions introduced in the release.
+When Sentry is notified of a release and the commits associated with it, we can help you pinpoint the commit that likely caused an issue - a _suspect commit_. This information helps your team resolve issues quickly because both suspect commits and suggested assignees will display on the Issues page in sentry.io.
 
-![](releases-overview.png)
+After notifying Sentry of a release, we can tie together the commits in the release, including:
+- Files touched by those commits
+- Files observed in the stack trace
+- Authors of those files
+- The issue number in a commit message
 
-### Install Repository Integration {#install-repo-integration}
+In addition, we can automatically assign or alert the issue owner, based on your organization's [ownership rules](/product/error-monitoring/issue-owners/).
 
-This step is optional - you can manually supply Sentry with your own commit metadata if you wish. Skip ahead to [this section](#alternatively-without-a-repository-integration) to learn how to do this.
+### Install a Repository Integration
 
-Using one of Sentry's repository integrations (such as GitHub, GitLab, Bitbucket, and so forth) is the easiest way to connect your commit metadata to Sentry. For a list of available integrations, go to Organization Settings > Integrations.
+Notifying Sentry of a release and providing commit metadata can be done most easily using one of our repository integrations. We also support a [manual process](/product/releases/manual-setup/without-repo) if you either don’t want Sentry to connect to your repository or you’re using an unsupported repository provider or VCS.
 
-<Note>
+<Alert level="warning" title="Required Permissions">
 
-You need to be an Owner or Manager of your Sentry organization to set up or configure an integration. Read more about [roles in Sentry](/product/accounts/membership/).
+To set up or configure an integration, you need to be an Owner or Manager of your Sentry organization. Learn more about [roles in Sentry](/product/accounts/membership/).
 
-</Note>
+If you’re linking a GitHub repository, you must have Admin or Owner permissions for the repository, and Sentry must be an authorized GitHub app in your [GitHub account settings](https://github.com/settings/applications).
```

Comment:
Do we allow abbreviations like "you're" in the new language guidelines? Would also apply to other instances in this page.

Reply:
We do - from our Writing Guide - "Contractions are easier to read and casual." We only avoid them when we wish to be assertive.

Reply:
Thanks, I'll study the guide.